modules/oxide/src/less/theme/components/pop/pop.less

```diff
@@ -63,15 +63,15 @@
   .tox-pop::before,
   .tox-pop::after {
     border-style: solid;
-    content: '';
+    content: 0;
```

Comment:
This looks wrong?

Reply:
reverted, issue caused by stylelint config previously